zuul/tenants.yaml

```diff
@@ -27,6 +27,7 @@
           - ansible-collections/cisco.ios
           - ansible-collections/cisco.iosxr
           - ansible-collections/cisco.nxos
+          - ansible-collections/community.general
```

Comment:
Do you want zuul to do the merge or still have humans manually do it.  If humans, this needs to move the line 95 below